arch/arm64/boot/dts/qcom/sdm660-xiaomi-jasmine.dts

```diff
@@ -0,0 +1,648 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2020, Alexey Minnekhanov <alexey.min@gmail.com>
+ * Copyright (c) 2021, Dang Huynh <danct12@riseup.net>
```

Comment:
I'm clearly not the author of jasmine dts, and Danct12 isn't too

Reply:
This dts was copied over from xiaomi-lavender (they are pretty closely related anyways) and I'm not entirely sure how copyright works in this sense.

Reply:
All DTS'es are quite similar, it's just hardware description file, what's to copyright there? Put your own.

Also recently they prefer to accept bindings with `// SPDX-License-Identifier: BSD-3-Clause` license